MeetingManager/MeetingManager.Infrastructure/Repositories/MeetingRepository.cs

```diff
@@ -78,6 +106,13 @@ private async Task<Meeting> getMeetingWithParticipants(int id)
                 .FirstOrDefaultAsync(m => m.Id == id);
         }
 
+        private async Task<List<Meeting>> setCache()
+        {
```

Comment:
create class InMemoryCache : IMemoryCache  and implement set and get methods.
get cahe data in service. remove all set and get data from repository